backend/src/main/java/com/capstone/moneytree/model/node/User.java

```diff
@@ -113,4 +94,13 @@ public void owns(Stock stock) {
         }
         this.getStocks().add(stock);
     }
-}    
\ No newline at end of file
+
+    public static User sanitizeUser(User user) {
+        User sanitizedUser = User.builder().firstName(user.getFirstName()).lastName(user.getLastName())
```

Comment:
Minor comment here: could you put an attribute per line? So that it is better to read for the future. For instance:
```
User sanitizedUser = User.builder().
                                         .firstname(...)
                                         .lastName(...)
                                         ...
                                        .build();
```